app/forms/journeys/additional_payments_for_teaching/induction_completed_form.rb

```diff
@@ -0,0 +1,27 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class InductionCompletedForm < Form
+      attribute :induction_completed
+
+      validates :induction_completed, presence: {message: ->(object, _) { object.i18n_errors_path("select_yes_if_completed") }}
+
+      def initialize(claim:, journey:, params:)
+        super
+
+        self.induction_completed = permitted_params.fetch(:induction_completed, claim.eligibility.induction_completed)
+      end
+
+      def save
+        return false unless valid?
+
+        update!({eligibility_attributes: {induction_completed:}})
+      end
+
+      private
+
+      def i18n_form_namespace
+        "induction_completed"
+      end
```

Comment:
This method is no longer required to override as it's clearly not needed on every form that follows this conventional naming; it's safe to keep, and we can remove it as part of another PR if that's easier.